### ISSUE TITLE
The mistake about getting AJI?

### ISSUE THREAD

**Issue by Lewislou** [Bingchao-Zhao/Triple_U-net#1]
Hi, I am curious about the AJI calculation of prediction images and the ground truth. 
From the code, looks like you are using binary predictions with binary masks to calculate the AJI. 
Would this kind of calculation wrong because AJI calculation needs instance labels?

**Comment by caodoanh2001** [comment]
Agree. Seems like the authors using only nuclei binary masks to compute AJI. It is not correct.

**Comment by caodoanh2001** [comment]
Hi, I figured it out.

Actually the authors did not use the binary nuclei map to compare with binary GT masks.
But, the predicted nuclei map was binary ones. They were then performed instance segmentation from the binary maps by the function `label` from

https://github.com/Bingchao-Zhao/Triple_U-net/blob/44a50ab74acc78e69d8b8b59d4d91533e1d7d21d/source/metrics.py#L5

This function was performed in `get_fast_aji` function to compare them to GT instance map.

https://github.com/Bingchao-Zhao/Triple_U-net/blob/44a50ab74acc78e69d8b8b59d4d91533e1d7d21d/source/metrics.py#L26

I think it is clear now.
